libsawtooth/src/client/rest.rs

```diff
@@ -358,11 +358,16 @@ impl Into<ClientBlockHeader> for BlockHeader {
 /// Used for deserializing error responses from the Sawtooth REST API.
 #[derive(Debug, Deserialize)]
 struct ErrorResponse {
+    error: ErrData,
```

Comment:
Can you provide a bit more context in the commit message about why this change is necessary? Was the old struct incorrect for parsing the error response?

Reply:
Updated commit message, previous struct wasn't deserializing the REST API error messages properly so it needed the additional struct